cls/SourceControl/Git/Settings.cls

```diff
@@ -133,6 +169,12 @@ Method %Save() As %Status
     // update value of basicUserMode to reflect the updated setting for basicMode
     set ..userBasicMode = ##class(SourceControl.Git.Utils).UserBasicMode()
     
+    set @storage@("settings","decomposeProductions") = ..decomposeProductions
+    if (..decomposeProductions & ($data(..Mappings("PTD"))<10)) {
+        set ..Mappings("PTD","*") = "ptd/" // configure a default mapping for PTD items
```

Comment:
Question: do we allow a mapping other than "ptd" for these? Or is the folder hardcoded anywhere?

Reply:
I've been testing with the PTD items mapped to a different subdirectory and all is working as expected.